xql/README.md

```diff
@@ -136,4 +136,55 @@ _Updated on 2024-01-08_
 7. [ ] **Aliases**: Add support to alias while querying.
 8. [ ] **Join Operations**: Support joining tables and apply query.
 9. [ ] **Nested Queries**: Add support to write nested queries.
-10. [ ] **Custom Aggregate Functions**: Support custom aggregate functions
\ No newline at end of file
+10. [ ] **Custom Aggregate Functions**: Support custom aggregate functions
+
+# `weather-lm` - Querying Xarray Datasets with SQL
```

Comment:
nit: change the description ... its should be "Querying weather data using Natural Language prompts"

Reply:
Done.